src/main/java/org/junit/runners/Parameterized.java

```diff
@@ -23,34 +23,55 @@
  * When running a parameterized test class, instances are created for the
  * cross-product of the test methods and the test data elements.
  * <p>
- * For example, to test a Fibonacci function, write:
+ * For example, this class has a method that adds two integers and returns the sum.
  * <pre>
+ * &#047;* This is the class being tested. *&#047;
```

Comment:
How about simply "Example class that adds two integers"? You already implied it was the code being tested above